clients/fides-js/src/components/fides.css

```diff
@@ -485,7 +485,7 @@ div#fides-consent-content .fides-modal-description {
   justify-content: center;
 }
 
-.fides-modal-container .fides-button-group-brand {
+.fides-modal-footer .fides-button-group-brand {
```

Comment:
`.fides-modal-container` doesn't exist when things are embedded, so let's target the modal footer class instead.

Reply:
Since this component exists in both the modal and the embed, is it worth trying to rename it to something more generic or would that break too much stuff?

Reply:
which component are you referring to, the footer or the brand?

Reply:
Sorry, the modal footer, that is.  Or anything else that's getting used here that has `modal` in the name, I suppose.

Reply:
Yeah, I think eventually we'll want to rename everything as "layer1" and "layer2" instead for that reason. I don't think this now is the time to do that. 👍